openedx_filters/learning/filters.py

```diff
@@ -442,3 +442,29 @@ def run_filter(cls, block, context):
         """
         data = super().run_pipeline(block=block, context=context)
         return data.get("block"), data.get("context")
+
+
+class CourseEnrollmentQuerysetRequested(OpenEdxPublicFilter):
+    """
+    Custom class used to filter user's course enrollments by site, when a request is made by the user.
```

Comment:
Great description! The issue is that not many people will use this filter as described here. That type of documentation goes to the pipeline steps. I suggest something like: 
```suggestion
    Custom class used to create course enrollments queryset filters and its custom methods.
```
This following the other filters docstrings.

Reply:
Thanks @mariajgrimaldi, I modified this part.